web/gds-admin-ui/src/utils/index.js

```diff
@@ -1,6 +1,29 @@
+import config from '../config';
+import { ENVIRONMENT } from '../constants';
+
 export * from './array';
 
 
+function defaultEndpointPrefix() {
+    if (config.GDS_API_URL) {
+        return config.GDS_API_URL
+    }
+
+    switch (process.env.NODE_ENV) {
+        case ENVIRONMENT.DEV:
+            return "http://localhost:4434/v2"
+        case ENVIRONMENT.PROD:
+            return "https://api.admin.vaspdirectory.net"
```

Comment:
Should be `https://api.admin.vaspdirectory.net/v2` - but actually we'll need to switch on `IS_TESTNET` here. e.g.

```
case ENVIRONMENT.PROD:
    if (config.IS_TESTNET) {
      return "https://api.admin.trisatest.net/v2";
   } else {
     return "https://api.admin.vaspdirectory.net/v2";
   }
```